extensions/elixir/Cargo.toml

```diff
@@ -13,4 +13,4 @@ path = "src/elixir.rs"
 crate-type = ["cdylib"]
 
 [dependencies]
-zed_extension_api = "0.0.6"
+zed_extension_api = "0.1.0"
```

Comment:
Let's stay on v0.0.6 for now, since we don't need anything in the new API version and it will introduce an unneeded restriction on the current Preview version of Zed in order to get this fix.

(I would push to the branch myself, but GitHub does not allow pushing to branches created from forks in organizations).

Reply:
Sure!